backend/controllers/cellMove/cellMovePrisonerSearch.js

```diff
@@ -0,0 +1,58 @@
+const { alertFlagLabels, cellMoveAlertCodes } = require('../../shared/alertFlagValues')
+const { putLastNameFirst, formatLocation } = require('../../utils')
+
+module.exports = ({ prisonApi }) => async (req, res) => {
+  const {
+    user: { activeCaseLoad },
+  } = res.locals
+  const { keywords } = req.query
+
+  if (!keywords) {
+    const firstCall = keywords === undefined
```

Comment:
You've already got the perfect name for this, _hasSearched_

Reply:
hmm.. hasSearched currently means actually sent a search to the server - but I prob should change it